frontend/src/components/Navbar.vue

```diff
@@ -1,28 +1,48 @@
 <script setup lang="ts">
     import type {MenuItem} from 'primevue/menuitem';
-    import {useRouter} from 'vue-router';
+    import {useRouter, useRoute} from 'vue-router';
     import LanguageSelectBox from '@/components/LanguageSelectBox.vue';
 
     const router = useRouter();
+    const route = useRoute();
 
-    const items: MenuItem[] = [
+    const items: (MenuItem & { path: string })[] = [
```

Comment:
GPT did not understand the assignment 😅
No need for the `path` option here, it's not standard in PrimeVue.

We handle navigation using the `command` arrow function.